pkg/utils/helpers.go

```diff
@@ -0,0 +1,54 @@
+package utils
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/projectdiscovery/gologger"
+	"github.com/projectdiscovery/pdtm/pkg"
+)
+
+// returns a callback function and when it is executed returns a version string of that tool
+func GetVersionCheckCallback(toolName string, output io.Writer) func() {
+	return func() {
+		tools, err := FetchToolList()
+		if err != nil {
+			gologger.Error().Msg(err.Error())
```

Comment:
@RamanaReddy0M , this callback will not be used in flag hence I think it would be better that callback returns a version string instead of writing it to writer
```go
// returns a callback function and when it is executed returns a version string of that tool
func GetVersionCheckCallback(toolName string) func() string    { xxx }
```